pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md

```diff
@@ -8,6 +8,26 @@ model: /mesosphere/dcos/services/elastic/data.yml
 render: mustache
 ---
 
+# Release notes for Elastic Service version 3.1.0-7.4.1
+
+## Upgrades
+
+-   Upgraded Elasticsearch and Kibana to [7.4.1](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/release-notes-7.4.1.html)
+-   Upgraded SDK version from `0.56.3` to `0.57.3`. ([#70](https://github.com/mesosphere/dcos-elastic-service/pull/70))
+
+    Please take a look at the following SDK change logs for more information:
+
+    -   [0.56.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.3)
+    -   [0.57.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+    -   [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
+    -   [0.57.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.2)
+    -   [0.57.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.3)
+    
+## Bug Fixes
+
+-   Fixed Kibana Dashboard issue. ([#72](https://github.com/mesosphere/dcos-elastic-service/pull/72)
+
+
 # Release notes for Elastic Service version 3.0.0-7.3.2
```

Comment:
Why are we including release notes for a version which we do not offer or support?

Reply:
Hi Sarah, 
We Does support this version as well but for an information we are keeping just previous version as well in release notes!

Reply:
I have added a note to the Release Notes: "Although version 3.0.0-7.3.2 is no longer supported, the following changes were implemented in it. You will need to be aware of these changes when you upgrade to the current version, 3.1.0-7.4.1."

I also added a link to the Updates page from the Release Notes.